packages/app/src/background/services/credentials/index.ts

```diff
@@ -43,22 +53,90 @@ export default class VerifiableCredentialsService implements IBackupable {
     return VerifiableCredentialsService.INSTANCE;
   }
 
-  addVerifiableCredential = async (serializedVerifiableCredential: string): Promise<boolean> => {
+  addVerifiableCredentialRequest = async (serializedVerifiableCredential: string): Promise<void> => {
+    await deserializeVerifiableCredential(serializedVerifiableCredential);
```

Comment:
I guess we can remove this function from here now, since its redundant now, if we are already verifying the params we get from the user via the middleware `validateVerifiableCredential`

Reply:
I think we still need it because it can be called from dapp and we need to validate/sanitize user input.

Reply:
So it would make sense to call `await valdiateVerifiableCredential`.

Reply:
Updated to use the validation function, see below